CWE/events/usa_marshall_plan.txt

```diff
@@ -128,22 +128,22 @@ country_event = {
     name = "Inject the cash into the National Bank"
 	prestige = 100
 random_pop = { limit = { location = { is_capital = yes } type = capitalists } money = 1000000 }
-	ai_chance = { factor = 1 }
+	ai_chance = { factor = 0.5 }
```

Comment:
This should be factor = 1 because the AI will get their own treasury cash boost through another event. The capital injection into pops is to stimulate the economy